src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_command_type.py

```diff
@@ -54,7 +56,16 @@ def _execute_command(kwargs):
         try:
 
             def get_token(server, resource, scope): # pylint: disable=unused-argument
-                return Profile().get_login_credentials(resource)[0]._token_retriever() # pylint: disable=protected-access
+                try:
+                    return Profile().get_login_credentials(resource)[0]._token_retriever() # pylint: disable=protected-access
+                except adal.AdalError as err:
+                    #pylint: disable=no-member
+                    if (hasattr(err, 'error_response') and
+                            ('error_description' in err.error_response)
+                            and ('AADSTS70008:' in err.error_response['error_description'])):
```

Comment:
If the above three lines can be arranged in this form, it will be more readable:
```
if (hasattr(err, 'error_response') and
    ('error_description' in err.error_response) and
    ('AADSTS70008:' in err.error_response['error_description'])):
```